src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -439,13 +439,27 @@ public class AgentKnobs
             new EnvironmentKnobSource("AGENT_USE_NODE"),
             new BuiltInDefaultKnobSource(string.Empty));
 
+        public static readonly Knob ProcessHandlerSecureArguments = new Knob(
+            nameof(ProcessHandlerSecureArguments),
+            "",
```

Comment:
I'd suggest to not leave description empty

Reply:
Fixed, thanks!